lib/assets/javascripts/twine.js.coffee

```diff
@@ -276,6 +276,7 @@ Twine.bindingTypes =
 
 setupAttributeBinding = (attributeName, bindingName) ->
   booleanAttribute = attributeName in ['checked', 'disabled', 'readOnly']
+  fireEvent = true if attributeName == 'checked'
```

Comment:
To avoid this being `undefined`, you can write `fireEvent = (attributeName == 'checked')`